src/main/java/frc/robot/Robot.java

```diff
@@ -145,5 +146,7 @@ private void configureButtonBindings() {
         driveController.getButton(Button.kB).whenHeld(intake.discharge());
         driveController.getButton(Button.kX).whenHeld(controlPanel.spinForwards());
         driveController.getButton(Button.kY).whenHeld(controlPanel.spinBackwards());
+        copilotController.getButton(Button.kBumperRight).whenHeld(intake.deployPiston());
```

Comment:
Instead of mapping these two commands we should map the command group we created. Talk to Mason about what button he wants to use